src/context/EarnManagerProvider/providers/yearn/components/YearnManager/YearnDeposit.tsx

```diff
@@ -41,26 +41,105 @@ export const routes = [
   { path: DepositPath.ApproveSettings, label: 'Approve Settings' },
   { step: 2, path: DepositPath.Confirm, label: 'Confirm Deposit' },
   { path: DepositPath.ConfirmSettings, label: 'Confirm Settings' },
-  { step: 3, path: DepositPath.Broadcast, label: 'Broadcast' }
+  { step: 3, path: DepositPath.Status, label: 'Status' }
 ]
 
 type YearnDepositProps = {
   api: YearnVaultApi
 }
 
+type YearnVault = {
+  apy: string
+}
+
+type EstimatedGas = {
+  estimatedGasCrypto?: string
+}
+
+type YearnDepositValues = DepositValues & EstimatedGas
+
+export type YearnDepositState = {
+  vault: YearnVault
+  userAddress: string | null
+  approve: EstimatedGas
+  deposit: YearnDepositValues
+  loading: boolean
+}
+
+const initialState = {
+  vault: { apy: '0' },
+  userAddress: null,
+  loading: false,
+  approve: {} as EstimatedGas,
+  deposit: {} as YearnDepositValues
```

Comment:
I don't think the `as` is required here if you specific the type of the parent object per my suggestion.

Reply:
ahh thats much nice than the typecast